packages/core-components/api-report.md

```diff
@@ -1471,5 +1471,6 @@ export type WarningPanelClassKey =
 // src/components/DependencyGraph/types.d.ts:26:9 - (ae-unresolved-link) The @link reference could not be resolved: The package "@backstage/core-components" does not have an export "DependencyNode"
 // src/components/TabbedLayout/RoutedTabs.d.ts:9:5 - (ae-forgotten-export) The symbol "SubRoute_2" needs to be exported by the entry point index.d.ts
 // src/components/Table/Table.d.ts:20:5 - (ae-forgotten-export) The symbol "SelectedFilters" needs to be exported by the entry point index.d.ts
-// src/layout/ErrorBoundary/ErrorBoundary.d.ts:8:5 - (ae-forgotten-export) The symbol "SlackChannel" needs to be exported by the entry point index.d.ts
+// src/layout/ErrorBoundary/ErrorBoundary.d.ts:14:5 - (ae-forgotten-export) The symbol "ErrorBoundaryFallbackProps" needs to be exported by the entry point index.d.ts
```

Comment:
Forgot to export this type

Reply:
It is exported though:

https://github.com/backstage/backstage/pull/18395/files#diff-a47d61199cf24044e34058af37b305a88b37910a7526ca7bcb36d563d72c15dbR33-R37

I'm not sure exactly what these messages mean here. I even import it into the tests:

https://github.com/backstage/backstage/pull/18395/files#diff-97361230c144e4a024c26e34c8f650bbfc98b57b58683e85aeb0fec60bc4949bR19-R22